src/components/input/index.css

```diff
@@ -7,3 +7,51 @@ input with validation
 input with icon prefix
 
 */
+.input-wrapper {
+  display: inline-flex;
+  flex-direction: column;
+  margin: 0.5rem;
+  position: relative;
+  gap: 0.4rem;
+}
+.input-wrapper .error-msg {
```

Comment:
I think we should give at least a line space between two declarations. As it increases readability.

Reply:
good to have, so will skip for now